plugins/modules/docker_stack.py

```diff
@@ -57,6 +57,12 @@
         current stack definition.
     type: bool
     default: false
+  detach:
+    description:
+      - If false will add the C(--detach=false) option to the C(docker stack deploy) command.
+        This will have docker exit immediately instead of waiting for the stack services to converge.
```

Comment:
The way I understand the description of `--detach`, it sounds that `--detach=true` (the default) is "Exit immediately instead of waiting for the stack services to converge". Are you sure that `--detach=false` means that Docker exits immediately?

Reply:
Setting `--detach=false` means Docker waits for all tasks to converge before exiting. This implies that Docker does not detach and will block until the services reach a `running` state or are `healthy` if a health check is configured. This ensures that the deployment is stable before the command completes.

On the other hand, `--detach=true` (the default) means Docker exits immediately after starting the services, without waiting for them to converge.

So, `--detach=false` is indeed what makes Docker wait for the tasks to converge before exiting.

Reply:
Ok, in that case you need to update the description. Right now it says the opposite :)